File: Web2 Store/frontend/src/models/item.ts
export interface Item{
    id: number,
    quantity: number,
    articleName: string,
}
export interface HistoryItem{
    id: number,
    quantity: number,
    articleName: string,
}
export interface ActiveItem{
    id: number,
    quantity: number,
    articleName: string,
}
export interface MakeItem{
    quantity: number,
    articleId: number,
}

File: Web2 Store/Backend/Store.Core/Models/Item.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Store.Core.Models
{
    public class Item
    {
        [Required]
        [Key]
        public int Id { get; set; }
        [Required]
        [Range(1, int.MaxValue)]
        public int Quantity { get; set; }
        [Required]
        public string ArticleName { get; set; }
        [Required]
        public int OrderId { get; set; }
        public Order Order { get; set; }
        [Required]
        public int ArticleId { get; set; }
        public Article Article { get; set; }
    }
}
